packages/block-editor/src/components/block-icon/stories/index.story.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Internal dependencies
+ */
+import BlockIcon from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { paragraph } from '@wordpress/icons';
+
+const meta = {
+	title: 'BlockEditor/BlockIcon',
+	component: BlockIcon,
+	parameters: {
+		docs: { canvas: { sourceState: 'shown' } },
```

Comment:
It is a good idea to add a description of this component via the `parameters.docs.description.component` field.